### PR TITLE
Upgrade to image-to-ascii@3.0.0

### PR DESCRIPTION
**Pull request by IonicaBizau**
:warning: This is **not** tested ...but hopefully it works :grin:

---

 - :arrow_up: Upgraded to image-to-ascii@3.0.0
 - :fire: Removed `temp` as dependency, since it is not used anymore.
 - :rocket: Use the buffer object to convert the image in text output.

### CODE REVIEW

**Comment by IonicaBizau** [comment]
:bulb: For reference, this is what it was changed in 3.0.0: https://github.com/IonicaBizau/image-to-ascii/pull/25

**Comment by rgbkrk** [comment]
Way slicker.

**Comment by rgbkrk** [comment]
I'll merge and we'll find out if it's working!

File: index.js [hunk 1]
```diff
@@ -162,19 +161,9 @@ function main(c) {
...
-                const decodedData = new Buffer(data['image/png'], 'base64');
-                const writer = fs.createWriteStream(info.path);
-                writer.end(decodedData);
-                writer.on('finish', () => {
-                  imageToAscii(info.path, (imErr, converted) => {
-                    console.log(imErr || converted);
-                  });
-                });
+              const decodedData = new Buffer(data['image/png'], 'base64');
+              imageToAscii(info.path, (imErr, converted) => {
```
Comment (rgbkrk) [inline index.js]: where is info defined?

Comment (IonicaBizau) [inline index.js]: @rgbkrk I'm terrible when writing code without testing it. :joy:

**Pull request merged**